pkgs/development/python-modules/weasyprint/default.nix

```
@@ -0,0 +1,68 @@
+{ lib, isPy3k, fetchPypi, buildPythonPackage
+, glib, pango, fontconfig
+, cffi, pyphen, cairosvg, html5lib, tinycss2, cairocffi, cssselect2
+, pytestcov, pytestrunner, pytest-flake8, pytest-isort }:
+
+buildPythonPackage rec {
+  pname = "weasyprint";
+  version = "44";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "WeasyPrint";
+    sha256 = "0f5bzwac1c4wgsscdv6d2d08n6g3j7pmj76ja5q31v40vzkz96vr";
+  };
+
+  patches = [
+    ./ld-library-path.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace weasyprint/fonts.py \
+      --subst-var-by libpangoft2-1-0 ${lib.makeLibraryPath [ pango ]}/libpangoft2-1.0.so \
```

```suggestion
      --subst-var-by libpangoft2-1-0 ${lib.makeLibraryPath [ pango ]}/libpangoft2-1.0.${stdenv.hostPlatform.extensions.sharedLibrary} \
```
Not sure about the advantage of using `makeLibraryPath`.

---

It seems better to use for multiple outputs, like for `fontconfig` at least, since `lib/` is within `fontconfig.lib`.